src/Native/Runtime/CMakeLists.txt

```diff
@@ -114,10 +118,15 @@ else()
     unix/HardwareExceptions.cpp
     unix/UnixContext.cpp
     unix/UnixNativeCodeManager.cpp
-    ../libunwind/src/Unwind-EHABI.cpp
```

Comment:
This change should not be needed. We should not be building Full runtime flavor for WASM. We should be building Portable flavor only.

Reply:
Done.

Reply:
Removed